utilities/pipelines/dependencies/constructs/StoreVhdToStorage/deploymentScripts/Start-ImageTemplate.ps1

```diff
@@ -0,0 +1,79 @@
+<#
+    .SYNOPSIS
+    Create artifacts from a given image template
```

Comment:
Why do we call it `artifacts`? If I'm not mistaken, the output will be a new Image Definition `version`. To me, artifacts sounds a bit generic ^^

Reply:
Or call it `image artifacts`

Reply:
Replaced everywhere